shared/app/tab-bar/container.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import {connect} from 'react-redux'
+import TabBarRender from './index.render'
+
+import type {TypedState} from '../../constants/reducer'
+
+const mapStateToProps = (state: TypedState) => ({
```

Comment:
Could you please expand on this? I'm not sure I understand. What would the selector be for?